### DISCUSSION TITLE
Nvim copilot has no response

### DISCUSSION THREAD

**Discussion by ianderrington** [community/community#10308] [Copilot Conversations]
Hi All, First, great work with this. It is exciting to know that we can code much faster with this tool!

Importantly though, I recently updated to a new computer (to M1Max macbook) and previously copilot worked, now it doesn't. 

So I reinstalled nvim and followed the instructions [here](https://github.com/github/copilot.vim#getting-started) to re-install it using github option. 

Upon starting nvim, and typing `:Copilot setup` there is no response that I can see. 

Are there any suggestions you might have to remedy this issue?

**Comment by sudoFerraz** [comment]
Experiencing the same issue
